dependency_support/pip_requirements.in

```diff
@@ -13,6 +13,11 @@ pyyaml==6.0.1
 # We build most of z3 ourselves but building python is really complicated. Just
 # use pypi version
 z3-solver==4.14.0.0
+pytest==8.2.2
+cocotb==1.9.0
+cocotbext-axi==0.1.24
```

Comment:
I don't think this is available in our internal repository, so it would be better to untie this dependency for now and remove the associated script from the PR and import them separately.